core/src/main/java/com/orange/ouds/core/component/OudsControlItem.kt

```diff
@@ -97,18 +97,18 @@ internal fun OudsControlItem(
         val trailingElement: (@Composable () -> Unit)? = if (reversed) indicator else itemIcon
 
         val filteredModifier = modifier.filter { it !is EdgeToEdgePaddingElement }
-        Column(
+        Box(
             modifier = filteredModifier
                 .height(IntrinsicSize.Min)
                 .heightIn(min = controlItemTokens.sizeMinHeight.dp)
                 .widthIn(min = controlItemTokens.sizeMinWidth.dp)
```

Comment:
If I update the thickness of the `OudsHorizontalDivider` to `30.dp`, the height of the item increases.
I think we have to move `height`, `heightIn` and `widthIn` modifiers in the `Row` below.